app/ui/parse_resume.py

```diff
@@ -137,13 +113,57 @@ def match_to_jobs(self, skills, num_jobs=3, skills_per_job=10):
         assert isinstance(skills_per_job, int)
         assert skills_per_job > 0
 
-
-        print("input skills:{}".format(skills))
+        # This is the more specific names for the jobs (there is a 1-1 mapping)
+        self.topic_names = {1: u'Data Engineering (Big Data Focus)',
```

Comment:
wait are we back to hard-coding this? Why are we not treating this as a model resource?